### PR TITLE
Liquid changes

### PR DESCRIPTION
**Pull request by ghost**
:cl: Thefastfoodguy
spriteadd: Water now appears to be higher/lower depending on depth
tweak: Water now puts out fires on mobs
tweak: You only hear the drain noise when they're actually draining
code: Drains are now processed by the Liquids subsystem
add: You can now construct drains. Make the assembly in an autolathe.
/:cl:

### CODE REVIEW

**Comment by GuyonBroadway** [comment]
What happens if you make a pool of welding fuel or ethanol?

**Comment by GuyonBroadway** [comment]
Does that also put out fire is what i am asking.

**Comment by KayeArray** [comment]
Erm, you do realise layering is already changed right? 

layer = LERP(LOW_OBJ_LAYER, FLY_LAYER, depth / 15)

Though yours might look different, take some screenshots

**Comment by ghost** [comment]
@banthebantz i've split the one sprite into multiple sprites

**Comment by ghost** [comment]
Layering in action
capture
capture png3
capture2

**Comment by KayeArray** [comment]
looks nice

**Comment by ghost** [comment]
Thanks

File: hippiestation/code/game/objects/structures/drain.dm [hunk 1]
```diff
@@ -16,22 +16,22 @@
...
+		soundloop.start()
 		if(!L.is_static && L.viscosity)
 			var/chance = Clamp(50 / L.viscosity, 15, 100)
 			if(get_dist(src,L) < 2)
 				qdel(L)
 			else if(prob(chance))
 				step_to(L, src)
-	if(counter > 25 || !can_weld_shut)
-		STOP_PROCESSING(SSobj, src)
+	if(counter > 25 || welded)
```
Comment (KayeArray) [inline hippiestation/code/game/objects/structures/drain.dm]: the welded check is actually broken atm, move it to the top of the proc and have it return so welding works properly

Comment (ghost) [reply hippiestation/code/game/objects/structures/drain.dm]: Fixed

**Pull request closed without merging**